website/package.json

```diff
@@ -11,7 +11,7 @@
     "@hashicorp/react-case-study-slider": "6.0.0",
     "@hashicorp/react-code-block": "4.0.1",
     "@hashicorp/react-content": "7.0.1",
-    "@hashicorp/react-docs-page": "13.1.4",
+    "@hashicorp/react-docs-page": "13.1.7-alpha.9",
```

Comment:
Will bump this to a stable release once these PRs are finalized and https://github.com/hashicorp/react-components/pull/220 is merged and published.

Reply:
✅ Bumped to stable